examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -219,64 +223,89 @@ void loop()
   }
 
   // test for switch input
-  processSwitches();
+  check = false;
+  processSwitches(check);
 
   // bottom of loop()
 }
 
-void processSwitches(void) {
-  bool isSuccess = true;
-  for (byte i = 0; i < NUM_SWITCHES; i++) {
-    moduleSwitch[i].update();
-    if (moduleSwitch[i].changed()) {      
-      byte nv = i+1;
-      byte nvval = modconfig.readNV(nv);
-      bool state;
-
-      //DEBUG_PRINT(F("> Button ") << i << F(" state change detected"));
-      //DEBUG_PRINT(F(" NV Value = ") << nvval);
-
-      switch (nvval) {
-        case 0:
-          // ON and OFF
-          state = (moduleSwitch[i].fell());
-          //DEBUG_PRINT(F("> Button ") << i << (moduleSwitch[i].fell() ? F(" pressed, send state: ") : F(" released, send state: ")) << state);
-          epService.sendEvent(state, i);
-          break;
-
-        case 1:
-          // Only ON
-          if (moduleSwitch[i].fell()) {
-            state = true;
-            //DEBUG_PRINT(F("> Button ") << i << F(" pressed, send state: ") << state);
-            epService.sendEvent(state, i);
-          }
-          break;
-
-        case 2:
-          // Only OFF
-          if (moduleSwitch[i].fell()) {
-            state = false;
-            //DEBUG_PRINT(F("> Button ") << i << F(" pressed, send state: ") << state);
-            epService.sendEvent(state, i);
-          }
-          break;
-
-        case 3:
-          // Toggle button
-          if (moduleSwitch[i].fell()) {
-            switchState[i] = !switchState[i];
-            state = (switchState[i]);
-            //DEBUG_PRINT(F("> Button ") << i << (moduleSwitch[i].fell() ? F(" pressed, send state: ") : F(" released, send state: ")) << state);
+byte checkProduced(void)
+{
+  check = true;
+  byte pIndex = processSwitches(check);
```

Comment:
Call processSwitch(true) instead of using the global variable "check".
Get rid of the global variable "check".

Reply:
Yes, done that